### PR TITLE
MG-93 - Add more apache echarts

### PR DESCRIPTION
**Pull request by Musilah**
# What type of PR is this?

- [ ] Refactor
- [x] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update
- [ ] Dependency Update

## Which issue(s) does this PR fix/relate to?

- Related Issue #93 
- Resolves #

## Have you included tests for your changes?

- [ ] Yes
- [x] No, and this is why: 

## Did you document any new/modified functionality?

- [ ] Yes
- [x] No, and this is why: 

### Notes
The charts included are: 
- multiline chart
- multibar chart
- multi gauge
- step chart
- shared dataset bar chart
- double bar chart

### CODE REVIEW

File: ui/web/templates/charts/doublebarchartmodal.html [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+    id="doubleBarChartModal"
+    tabindex="-1"
+    role="dialog"
+    aria-labelledby="doubleBarChartModalLabel"
+    aria-hidden="true"
+  >
+    <div class="modal-dialog modal-dialog-centered">
+      <div class="modal-content">
+        <div class="modal-header">
+          <h5 class="modal-title" id="doubleBarChartModalLabel">Time series bar chart</h5>
```
Comment (1998-felix) [inline ui/web/templates/charts/doublebarchartmodal.html:17]: The name on thre title of the modal and on the card do not match. Should they not be the same?

File: ui/web/templates/charts/multigaugemodal.html [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+    id="multiGaugeModal"
+    tabindex="-1"
+    role="dialog"
+    aria-labelledby="multiGaugeModalLabel"
+    aria-hidden="true"
+  >
+    <div class="modal-dialog modal-dialog-centered">
+      <div class="modal-content">
+        <div class="modal-header">
+          <h5 class="modal-title" id="multiGaugeModalLabel">Time series line charts</h5>
```
Comment (1998-felix) [inline ui/web/templates/charts/multigaugemodal.html:17]: Same as above.

**Review by 1998-felix — CHANGES_REQUESTED** [review]

**Review by 1998-felix — COMMENTED** [review]
image
The multigauge modal and the double bar chart.

File: ui/charts.go [hunk 3]
```diff
@@ -16,69 +16,78 @@ func CreateItem() []Item {
...
-			Image:   "https://datavizproject.com/wp-content/uploads/types/Bar-Chart-Horizontal.png",
 			Widget:  "horizontalBarChart",
 		},
+		{
+			Title:   "Double Bar Chart",
+			Content: "This is a double bar chart",
+			Widget:  "doubleBarChart",
+		},
+		{
+			Title:   "mutliple line Chart",
```
Comment (1998-felix) [inline ui/charts.go:77]: Capitalize

**Comment by 1998-felix** [comment]
This pr removes thumbnails which will be returned by addressing issue: https://github.com/absmach/magistrala-ui/issues/149

**Review by dborovcanin — CHANGES_REQUESTED** [review]
Resizing the newly added charts does not work properly.

**Comment by ianmuchyri** [comment]
All these charts you are adding seem to also be in the e-charts library. Did you check there?

**Review by ianmuchyri — CHANGES_REQUESTED** [review]

**Review by ianmuchyri — APPROVED** [review]

**Review by 1998-felix — APPROVED** [review]

**Review by dborovcanin — APPROVED** [review]

**Pull request merged**